app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -146,9 +147,18 @@ class Settings private constructor(
 
     // If any of the prefs have been modified, quit displaying the fenix moved tip
     fun shouldDisplayFenixMovingTip(): Boolean =
-        preferences.getBoolean(appContext.getString(R.string.pref_key_migrating_from_fenix_nightly_tip), true) &&
-            preferences.getBoolean(appContext.getString(R.string.pref_key_migrating_from_firefox_nightly_tip), true) &&
-            preferences.getBoolean(appContext.getString(R.string.pref_key_migrating_from_fenix_tip), true)
+        preferences.getBoolean(
```

Comment:
Looks like this is just a whitespace change, should it be removed?

Reply:
I agree it looks a bit ugly but this is what the autoformatter with the Kotlin style guide wants 🤷‍♀️ Don't have strong preferences but anytime you reformat this file you'd run into this